stable/rethinkdb/templates/rethinkdb-cluster-stateful-set.yaml

```diff
@@ -64,17 +72,9 @@ spec:
                 fieldRef:
                   fieldPath: metadata.name
           livenessProbe:
-{{ toYaml .Values.cluster.livenessProbe | indent 12 }}          
+{{ toYaml .Values.cluster.livenessProbe | indent 12 }}
           readinessProbe:
 {{ toYaml .Values.cluster.readinessProbe | indent 12 }}
-            exec:
```

Comment:
This section should have been removed when the `cluster.readinessProbe` config value was added.